test/built-ins/JSON/stringify/replacer-function-object-deleted-property.js

```diff
@@ -0,0 +1,41 @@
+// Copyright (C) 2020 Alexey Shvayka. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+esid: sec-serializejsonproperty
+description: >
+  Replacer function is called on properties, deleted during stringification.
+info: |
+  JSON.stringify ( value [ , replacer [ , space ] ] )
+
+  [...]
+  12. Return ? SerializeJSONProperty(the empty String, wrapper).
+
+  SerializeJSONProperty ( key, holder )
+
+  1. Let value be ? Get(holder, key).
+  [...]
+  3. If ReplacerFunction is not undefined, then
+    a. Set value to ? Call(ReplacerFunction, holder, « key, value »).
```

Comment:
The real thing be tested here is whether step 5 & 6 occur prior to the operations nested in step 8.